src/librustc_mir/error_codes.rs

```diff
@@ -157,81 +157,6 @@ match x {
 See also the error E0303.
 "##,
 
-E0008: r##"
```

Comment:
Why has this been removed at all, as opposed to having `#### Note: this error code is no longer emitted by the compiler.` prepended to the description (like below)?

Reply:
The difference is that in those cases, the error codes has been exchanged for other error codes, but the examples and the textual description is still valid. However, in the case of E0008, there is no such error code and the text would need to be changed because it says things are not allowed but they are in fact allowed not. I tried to keep this text somehow but I found that it didn't really work well. Open to suggestions on how to fix this.

Reply:
My one concern is that users using an older version of the compiler, but viewing the error code documentation online for the current version of the compiler, will not be able to read a detailed description of the error code (though really this concern is more general, and applies also whenever an error code is significantly repurposed). (@matthewjasper's suggestion is fine for compiler developers, but doesn't help users in this situation.) It would be good if we could come up with a more principled solution, though @matthewjasper's suggestion suffices for now.

Reply:
@matthewjasper Re-added a line to `error_codes.rs` at the bottom though I'm not sure if you wanted the whole text as well. Can you please check if this is what you wanted?